ftest.sh

```diff
@@ -61,6 +61,12 @@ if [ -n "${3}" ]; then
     NVME_ARG="-n ${3}"
 fi
 
+# Number of times to repeat the tests run by launch.py
+TEST_REPEAT="1"
+if [ -n "${4}" ]; then
```

Comment:
Right.  But it becomes mandatory as soon as `${5}` is added just like the previously optional `$NVME_ARG` becomes mandatory as soon as you want a non-default repeat value.  This just goes on and on as more arguments are added.

Reply:
Due to the https://github.com/daos-stack/pipeline-lib/blob/d4d90d56c6eb4187b3e7710a0b1a1f685d21c32f/vars/parseStageInfo.groovy#L136 change (moving the adding of the `-n|--nvme` argument identifier into `parseStageInfo.groovy` from `ftest.sh`) this will probably require a coordinated landing - unless I add some extra handling into `ftest.sh` to support receiving a `--nvme` argument w/o the `-n|--nvme` identifier.  Thoughts?

Reply:
We can't/don't (generally) do "coordinated landing"s as it breaks everything in flight, requiring people to do another round of CI.  We introduce backward/forward compatibility instead.  Another plus for env. var.s.  :-)

Reply:
That's reasonable.  I've added code to handling interfacing with the current pipeline lib implementation - this should allow this change to land (once I revert the `Jenkinsfile` change) prior to the pipeline-lib change.  I did not make the `NVME_ARG/LAUNCH_OPT_ARGS` ftest.sh input an environment variable as that would require a coordinated change between pipeline-lib and daos, but at least the number of command line arguments for ftest.sh is not changing.

Reply:
The current build (https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-5678/5/pipeline) is verifying the behavior of the `Test-repeat*` pragma.  The next build will verify the (normal) behavior w/o the new pragma.